examples/AspNetCore/Startup.cs

```diff
@@ -111,6 +115,16 @@ public void ConfigureServices(IServiceCollection services)
 
                     break;
             }
+
+            // TODO: Add IServiceCollection.AddOpenTelemetryMetrics extension method
+            var providerBuilder = Sdk.CreateMeterProviderBuilder()
```

Comment:
until we add Ext.Hosting support, we can simply add the meterprovider as a singleton in the services, to ensure it gets disposed at end.